### PR TITLE
Auto deploy docs

### PR DESCRIPTION
**Pull request by harterrt**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- "4.1"
+install:
+- npm install gitbook-cli -g
+script:
+- gitbook install
+- gitbook build
+skip_cleanup: true
+deploy:
+  provider: pages
+  skip-cleanup: true
```
Comment (fbertsch) [inline .travis.yml]: Does this need `skip-cleanup: true` twice?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+- "4.1"
```
Comment (mreid-moz) [inline .travis.yml]: nit: indent list items here and below

Comment (harterrt) [reply .travis.yml]: Done.

File: .travis.yml (hunk 1 above)
Comment (harterrt) [reply .travis.yml]: Fixed.

**Comment by harterrt** [comment]
Argh - missed your commend on fqdn. Started https://github.com/mozilla/firefox-data-docs/pull/96 to fix that.

**Pull request merged**